docs/source/welcome/services/consultations.md

```diff
@@ -0,0 +1,8 @@
+(consultations)=
+
+# Consultations
```

Comment:
Do we want to add the office hours information? Could just pull this from the website? https://rc.northeastern.edu/support/consulting/